rpc/wrtc_base_stream.go

```diff
@@ -38,16 +41,21 @@ type webrtcBaseStream struct {
 func newWebRTCBaseStream(
```

Comment:
I'm inclined to leave a comment somewhere (but my guess was here was the best surfaced, but maybe RecvMsg?):

```
RSDK-4473: There are two ways a stream can be signaled that it should give up receiving a message:
- `ctx` is canceled due to a timeout or from a logical error.
- `channelCtx` is canceled when the underlying webrtc data channel connection experiences an error.
```

I don't feel great in my summarization of the former as a "logical error". It felt more appropriate than say "an error from the goutils grpc/webrtc code".

Reply:
Yeah good idea; left the comment you wrote out before the relevant `select` case; changed the wording slightly.